website/docs/docs/use-dbt-semantic-layer/avail-sl-integrations.md

```diff
@@ -1,31 +1,54 @@
 ---
 title: "Available integrations"
 id: avail-sl-integrations
-description: "Review a wide range of partners you can integrate and query with the dbt Semantic Layer."
+description: "Discover the diverse range of partners that seamlessly integrate with the powerful dbt Semantic Layer, allowing you to query and unlock valuable insights from your data ecosystem."
 sidebar_label: "Available integrations"
 ---
 
-:::info Coming soon
-The dbt Semantic Layer is undergoing a [significant revamp](https://www.getdbt.com/blog/dbt-semantic-layer-whats-next/), making it more efficient to define and query metrics.
+<VersionBlock firstVersion="1.6">
 
-**What’s changing?** The dbt_metrics package will be [deprecated](https://docs.getdbt.com/blog/deprecating-dbt-metrics) and replaced with [MetricFlow](/docs/build/about-metricflow?version=1.6), a new way framework for defining metrics in dbt.
+import NewSLChanges from '/snippets/_new-sl-changes.md';
 
-**What's new?**  Learn how to [Build your metrics](/docs/build/build-metrics-intro?version=1.6) using MetricFlow, one of the key components that makes up the revamped dbt Semantic Layer. It handles SQL query construction and defines the specification for dbt semantic models and metrics. 
-:::
+<NewSLChanges />
+
+Discover the diverse range of data applications that seamlessly integrate with the powerful dbt Semantic Layer, powered by MetricFlow &mdash; from business intelligence tools to notebooks, spreadsheets, data catalogs, and more. These integrations allow you to query and unlock valuable insights from your data ecosystem.
+
+Query dbt metrics via external integrations using the sophisticated [dbt Semantic Layer APIs](/docs/use-dbt-semantic-layer/sl-api-overview) (JDBC, GraphQL, and Discovery APIs). With these APIs, you can query metrics to avoid duplicative coding, optimize your development workflow, ensure data governance for company metrics, and guarantee consistency for data consumers.
+
+The dbt Semantic Layer integrations are capable of querying dbt metrics, importing definitions, surfacing the underlying data in partner tools, and more. Here are some of the integrations you can query from:
+
+- Tableau &mdash; 
```

Comment:
Tableau definitely isn't making it so we can remove

Reply:
We can also add "Any SQL developer tool that supports a generic driver (e.g., DataGrip)"

Reply:
Gsheets is a maybe. A beta would be exciting but I think it would be tough to get it all figured out by then. We can say "Coming soon!"

Reply:
done!